src/Google/Ads/GoogleAds/Util/Dependencies.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Ads\GoogleAds\Util;
+
+/**
+ * Utilities for dependencies of the Google Ads API PHP client library.
+ */
+class Dependencies
+{
+    /**
+     * Gets the grpc version installed by Composer from the specified file (default to
+     *  `composer.lock`. Returns null if that information cannot be found by any causes.
+     *
+     * @param string $fileName the file name to extract the grpc version from
+     * @return null|string the grpc version installed by Composer
+     */
+    public function getGrpcComposerVersion(string $fileName = 'composer.lock'): ?string
```

Comment:
Tests are the only code that will ever specify a different filename from the default, correct?